YT_CRUD/src/Controller/ArticleController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+  namespace App\Controller;
+
+  use Symfony\Component\HttpFoundation\Response;
+  use Symfony\Component\Routing\Annotation\Route;
+  use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+  use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+  class ArticleController extends Controller {
+    /**
+    * @Route("/")
+    * @Method({"GET"})
+    */
+    public function index() {
+      // return new Response('<html><body>Hello</body></html>');
+
+      $articles = ['Article1', 'Article2', 'Article3'];
+
+      return $this->render('articles/index.html.twig', array('articles' => $articles));
+    }
+  }
+
+
```

Comment:
@JanMaria Jeżeli dajesz puste linie pomiędzy kodem to tylko i wyłącznie pojedyncza pusta linia,